runelite-client/src/main/java/net/runelite/client/plugins/hunter/HunterPlugin.java

```diff
@@ -387,4 +420,21 @@ public void onConfigChanged(ConfigChanged event)
 			overlay.updateConfig();
 		}
 	}
+
+	@Subscribe
+	public void onGameStateChange(GameStateChanged event)
+	{
+		switch (event.getGameState())
```

Comment:
just clear them on hopping and login_screen, and make the birdhouse array final and initialize it right away